oat-sa/tao-core
# Feature/nex 88 integrate pull request ci

Brings Pull Request CI to this repository

How to validate it ?
This Pull Request should have 2 additionnal checks : 
 - the FE and BE tests that run against the pr head
  - the FE and BE tests that run against the branch
